pkg/webhook/resources/setting/validator.go

```diff
@@ -14,9 +14,6 @@ import (
 	"strings"
 
 	"github.com/longhorn/backupstore"
-	// Although we don't use following drivers directly, we need to import them to register drivers.
```

Comment:
please keep this, otherwise the backup validation would fail

Reply:
👍